rtsp.go

```diff
@@ -365,7 +373,11 @@ func newRTSPCamera(ctx context.Context, _ resource.Dependencies, conf resource.C
 		u:      u,
 		logger: logger,
 	}
-	err = rtspCam.reconnectClient()
+	codecInfo, err := modelToCodec(conf.Model)
```

Comment:
Can we please change `codecInfo` to be a struct member on `rtspCamera` rather than a perameter given that it never changes for the lifetime of an `rtspCamera` struct?

Reply:
I'd actually rather pass it around in the creator/reconfigure since it's only used there than have it on the struct, it's less state to check in other API calls in this driver.